cfme/tests/infrastructure/test_esx_direct_host.py

```diff
@@ -39,8 +39,11 @@ def pytest_generate_tests(metafunc):
 
         host = hosts[0]
         ip_address = resolve_hostname(host["name"])
-        endpoint = VirtualCenterEndpoint(credentials=host["credentials"], hostname=host["name"],
-                                         ip_address=ip_address)
+        endpoint = VirtualCenterEndpoint(
+            credentials=host["credentials"]["default"],
```

Comment:
**QUESTION** i presume the "default" was the missing bit?

Reply:
It required by new yaml structure